Update Smart_Serial_Connection.md (#4825)
SkylineCommunications/dataminer-docs: user-guide/Basic_Functionality/Elements/Adding_Elements/Smart_Serial_Connection.md

  To add another accepted IP address, below the box, click *Add*.

  > [!NOTE]

>   > This functionality can be disabled if *AllowedIPAddresses.Disabled* is set to “true” in the user settings of the smart-serial connection in the protocol.

Reviewer: The current phrasing suggests that it is possible to disable the functionality if that setting is set to true, but it isn't necessarily disabled based on the setting. If this isn't what you meant, I'd suggest rephrasing this like this:
```suggestion
  > This functionality can be disabled by setting *AllowedIPAddresses.Disabled* to “true” in the user settings of the smart-serial connection in the protocol.
```
